### ISSUE TITLE
open new wt.exe can't update user path

### ISSUE THREAD

**Issue by fcying** [microsoft/terminal#8542]
# Environment

```none
Windows build number: Microsoft Windows [10.0.19042.662]
Windows Terminal version (if applicable):1.4.3243.0(install by scoop)
commandline: cmd.exe
```

# Steps to reproduce
1> modify user path(add c:\test at last)
2> run wt.exe, echo %path%, can't see `c:\test`, run cmd.exe, can see path is update

# Expected behavior
path is update.

# Actual behavior
path is not update.

**Comment by zadjii-msft** [comment]
I'd bet this is one of those ithngs that regressed with #7243. I can't be sure what the issue we're using for tracking these is - @DHowett help me out? 

It's not #8217, but that one suggests
> Indeed. This was fixed in #7243, but I have been holding it back from Stable builds because it breaks WOW64 PowerShell. (#7418). 

Then got dup'ed to #7243.

**Comment by fcying** [comment]
thanks for the reply~
